.github/workflows/ci.yml

```diff
@@ -89,26 +89,13 @@ jobs:
         with:
           toolchain: stable
       - uses: Swatinem/rust-cache@a95ba195448af2da9b00fb742d14ffaaf3c21f43
-      - name: Set starknet-devnet-rs rev
-        id: devnet-rev
-        run: |
-          devnetrev=$(grep "DEVNET_REV=" ./scripts/prepare_for_tests.sh | awk -F '=' '{print $2}')
-          echo "devnet_rev=${devnetrev//\"}" >> $GITHUB_OUTPUT
-      - name: Cache starknet-devnet-rs
-        id: cache-devnet
-        uses: actions/cache@v3
         with:
-          path: ./crates/cast/tests/utils/devnet
-          key: starknet-devnet-rs-${{ steps.devnet-rev.outputs.devnet_rev }}
+          cache-directories: crates/cast/tests/utils/devnet
```

Comment:
how does this work? I can't see a cache ket, so is it somehow automatically generated? will this cache change when any of the other cached deps changes? will this change when we up the rev of devnet?

Reply:
We don't define keys for any other usages of `Swatinem/rust-cache` action so I think automatic generation should be fine. I'll test how it behaves when deps change and when devnet rev changes.

Reply:
ok, so there is automatic key generation, I assume based on hash? 
that would be great, I'd really like to avoid a situation where changing one of the deps forces rebuild of devnet (and vice-versa), as this compilation takes quite some time

Reply:
that's awesome, thanks for taking the time and checking this!